### PR TITLE
ESY-2578 Add basic explanation of unscheduled

### PR DESCRIPTION
**Pull request by sigridbra**

### CODE REVIEW

File: _includes/delete-payment-order-token.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% capture api_resource %}{% include api-resource.md %}{% endcapture %}
+{% assign token_field_name = include.token_field_name %}
+{% capture token_url %}
+    /psp/{{ api_resource }}/{{ token_field_name }}s/{{- page.payment_token -}}
+{% endcapture %}
+{% assign token_url=token_url | strip %}
+
+## Delete {{ token_field_name }}
+
+Payers should be able to delete payment tokens that are associated to
```
Comment (sigridbra) [inline _includes/delete-payment-order-token.md:10]: Should this text be differentiated from delete_token? 
We could potentially migrate all delete paymentOrder recurrenceTokens to use this file

**Comment by sigridbra** [comment]
I would like to add a section like [this ](https://github.com/SwedbankPay/developer.swedbankpay.com/blob/develop/_includes/recurring-card-payments.md#retrieve-recurrence-token)regarding how to retrieve the token, but I cant seem to find such a section for paymentOrder.  Maybe this should be done in a phase 2 of the documentation.

File: _includes/payment-order-unscheduled.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+{% capture features_url %}{% include documentation-section-url.md href='/features' %}{% endcapture %}
+
+## Unscheduled Purchase
+
+An `unscheduled purchase`, also called a Merchant Initiated Transaction (MIT),
+is a payment which uses a `unscheduledToken` generated through a previous payment in
+order to charge the same card at a later time. They are done by the merchant
+without the cardholder being present.
```
Comment (sigridbra) [inline _includes/payment-order-unscheduled.md:8]: I'll ask Per M if he has a good explanation on the difference between unscheduled and recur in line with the new guidelines from Acquiring. If so, this text might be changed a bit.

**Review by asbjornu — APPROVED** [review]

**Comment by asbjornu** [comment]
> I would like to add a section like [this ](https://github.com/SwedbankPay/developer.swedbankpay.com/blob/develop/_includes/recurring-card-payments.md#retrieve-recurrence-token)regarding how to retrieve the token, but I cant seem to find a fitting location for paymentOrder tokens. Maybe this should be done in a phase 2 of the documentation of unscheduled (and the updates to come regarding the future of recurring payments).

Since this is supposed to reach staging ASAP, I suggest we just merge this and that you submit a new PR with Per's new text and other potential improvements.

**Pull request merged**